src/sass/components/_scrollbars.scss

```diff
@@ -1,10 +1,15 @@
 // TODO: Prefixes
 
 ::-webkit-scrollbar {
-    width: 8px;
-    background-color: #fcfcfc;
+    width: 10px;
+    background-color: $scrollbar-color-bg;
     border: 1px solid #f9f9f9;
 }
 ::-webkit-scrollbar-thumb {
-    background-color: #ddd;
+    background-color: $scrollbar-color-thumb;
+}
+
+@mixin scrollbar {
+    scrollbar-width: auto;  // Unfortunately Firefox does not support exact pixel values
```

Comment:
Voting for `thin`. :slightly_smiling_face:

Reply:
Done.